packages/apex-node/package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "@salesforce/apex-node",
   "description": "Salesforce js library for Apex",
-  "version": "0.3.0",
+  "version": "0.4.0",
```

Comment:
we should not need to bump the version manually here with https://github.com/forcedotcom/salesforcedx-apex/pull/255/files